diff --git a/node_modules/brorand/index.js b/node_modules/brorand/index.js
index 9a0fff4..b5b53eb 100644
--- a/node_modules/brorand/index.js
+++ b/node_modules/brorand/index.js
@@ -45,10 +45,11 @@ if (typeof self === 'object') {
 
   // Safari's WebWorkers do not have `crypto`
   } else if (typeof window === 'object') {
-    // Old junk
-    Rand.prototype._rand = function() {
-      throw new Error('Not implemented yet');
-    };
+		Rand.prototype._rand = function _rand(n) {
+			var arr = new Uint8Array(n);
+			self.crypto.getRandomValues(arr);
+			return arr;
+		};
   }
 } else {
   // Node.js or Web worker with no crypto support
